vizro-core/examples/scratch_dev/app.py

```diff
@@ -1,60 +1,70 @@
-"""Dev app to try things out."""
+from typing import List, Literal
+
+from dash import html
 
 import vizro.models as vm
 import vizro.plotly.express as px
 from vizro import Vizro
+from vizro.models.types import ControlType
+
+df_gapminder = px.data.gapminder()
+
+
+class ControlGroup(vm.VizroBaseModel):
+    """Container to group controls."""
+
+    type: Literal["control_group"] = "control_group"
+    title: str
+    controls: List[ControlType] = []
 
-gapminder_2007 = px.data.gapminder().query("year == 2007")
+    def build(self):
+        return html.Div(
+            [html.H4(self.title), html.Hr()] + [control.build() for control in self.controls],
+            className="control_group_container",
```

Comment:
I don't know if we suggested this anywhere now, but the className is not required anymore, nor is any custom CSS, because the `H4` and the `Hr` element get automatically styled by out vizro-bootstrap theme :) 

```suggestion
```

Reply:
Ahhh nice, that is very good to hear 👍